passkey-api/next-auth.mdx

```diff
@@ -1,16 +1,56 @@
 ---
 title: Next.js with Auth.js
+description: Learn how to add passkeys to your Next.js app which already uses Auth.js for authentication.
 ---
 
-If you already have a Next.js app set up with Auth.js, you can easily add passkeys to your app using `@teamhanko/passkeys-next-auth-provider`.
+<Steps>
 
-Note that there's an [example repo](https://github.com/teamhanko/passkeys-example-next-auth) that showcases a Next.js app with all of the steps in this guide already implemented.
+<Step title="Install passkey provider">
 
-## Step one
+Once you’ve initialized your Next app with [Auth.js](https://authjs.dev/), install our passkey provider.
 
```

Comment:
Absolutely love this change. Having it immediately jump in is way, way better

Reply:
Also huge +1 for `<Step>` (i had no idea it existed) and putting the example repo as a card at the end!!